scale/job/management/commands/scale_post_steps.py

```diff
@@ -106,6 +106,7 @@ def _perform_post_steps(self, job_exe):
 
         job_interface = job_exe.get_job_interface()
         job_data = job_exe.job.get_job_data()
+        # TODO: fix this
```

Comment:
We need to pull the new logs (for the job task) in order to do stdout product registration. We still have jobs running that rely upon this.